xDrip/Scenes/Settings/BaseSettings/Cells/BaseSettingsDisclosureCell.swift

```diff
@@ -12,5 +12,6 @@ final class BaseSettingsDisclosureCell: UITableViewCell {
     func configure(mainText: String, detailText: String?) {
         textLabel?.text = mainText
         detailTextLabel?.text = detailText
+        accessoryType = .disclosureIndicator
```

Comment:
If this is hardcoded, it shouldn't go to `configure` method. Could you move it to either IB or some init/awake?